_beats/dev-tools/packaging/packages.yml

```diff
@@ -173,6 +173,11 @@ shared:
     extra_vars:
       repository: 'docker.elastic.co/beats'
 
+  - &docker_ubi_spec
+    extra_vars:
+      image_name: '{{.BeatName}}-ubi7'
+      from: 'registry.access.redhat.com/ubi7/ubi:7.7'
```

Comment:
Hi @graphaelli,
The base image to use is `registry.access.redhat.com/ubi7/ubi-minimal:7.7` unless there is technical issue. Did you try with this one before using `registry.access.redhat.com/ubi7/ubi:7.7`?

Reply:
@mikemadden42 mentioned that ubi-minimal was missing some things we need like `useradd` and `groupadd`, anything else worth mentioning?

Reply:
Logstash and Elasticsearch are using minimal image which is our requirement so I guess you can use useradd and groupadd or have a workaround if needed. However Beats should switch to minimal image for 7.6.0. If you're dependant of beats image for APM, that's fine as long as you can switch when Beats image will switch.

Reply:
Since beats needs to do the same, we can make the change there and then pull it into apm-server.  Elasticsearch simply installs `shadow-utils` per https://github.com/elastic/elasticsearch/blob/8c2dda90c0f01c7aa92ab82f0329e7b273c14fae/distribution/docker/src/docker/Dockerfile#L16-L19.  @mikemadden42 mind making the changes needed?